ambari-common/src/main/python/resource_management/libraries/execution_command/stack_settings.py

```diff
@@ -0,0 +1,109 @@
+#!/usr/bin/env python
+"""
+Licensed to the Apache Software Foundation (ASF) under one
+or more contributor license agreements.  See the NOTICE file
+distributed with this work for additional information
+regarding copyright ownership.  The ASF licenses this file
+to you under the Apache License, Version 2.0 (the
+"License"); you may not use this file except in compliance
+with the License.  You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+"""
+
+__all__ = ["StackSettings"]
+
+class StackSettings(object):
+  
+  # Stack related configs from stack's stack_settings.json
+  # TODO: these strings may be removed later
+  STACK_NAME_SETTING = "stack_name"
```

Comment:
Most of these constants are unused.  Why do we need them?

Reply:
Not sure, so temporarily keep them here, will remove them later.